Evaluation/Input/Content/Section1_Introduction.md

```
@@ -0,0 +1,29 @@
+
+Dextromethorphan is a widely used over-the-counter cough suppressant and a common ingredient of cold medicines marketed toward children and adults.
+
+Dextromethorphan is typically administered as its hydrobromide salt, which is considered a Biopharmaceutics Drug Disposition Classification System (BDDCS) class I drug with high solubility and permeability. After oral administration, dextromethorphan is rapidly absorbed. Dextromethorphan undergoes an extensive first-pass metabolism, predominately mediated by CYP2D6, reducing the bioavailability to 1%–2% in CYP2D6 extensive metabolizers (EMs) and 80% in CYP2D6 poor metabolizers (PMs). 
+
+The herein presented model building and evaluation report evaluates the performance of the PBPK model for Dextromethorphan in (healthy) adults published by Rüdesheim et al. 2025 ([Rüdesheim 2025](#5-References)). 
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)